pkg/metrics/metrics.go

```diff
@@ -40,9 +41,42 @@ var (
 			Help: "Number of successful gitops creation requests",
 		},
 	)
+
+	ControllerGitRequest = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "controller_git_request",
+			Help: "Number of git operation requests.  Not an SLI metric",
+		},
+		[]string{"controller", "tokenName", "operation"},
+	)
+
+	SecondaryRateLimitCounter = prometheus.NewCounterVec(
```

Comment:
Are we doing anything with this and the PrimaryRateLimitCounter counters yet? Or is this just adding the counters for now?

Reply:
we're just setting counters for now.  Eventually, they will be used in alerting.

Reply:
Sorry, what I mean is, I don’t see this or PrimaryRateLimitCounter set any where in the code? E.g checking for a secondary rate limit error